README.md

```diff
@@ -52,10 +52,18 @@ Then, we will work on BPMN extensions, library extension points, display options
 capacities.
 
 
-# Development
+# Contributing
 
-To build the project, see the [Contributing guide](CONTRIBUTING.md#Build) :slightly_smiling_face:
+To contribute to `bpmn-visualization`, fork and clone this repo locally and commit your code on a separate branch. \
+Please write tests for your code before opening a pull-request:
 
+```sh
+npm run test  # run all unit & e2e tests
```

Comment:
```suggestion
npm install
npm run test  # run all unit & e2e tests
```

Reply:
@csouchet @aibcmars do you agree with this suggestion? this is the last thing (apply it or not) which blocks the merge

Reply:
Here, we just want to remember the criteria for accepting pull requests (just run the test).
So, for me, how to install the dependency should not be in this section.

Reply:
This makes sense, I am ok with that.